linode/childaccount/datasource_test.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/linode/terraform-provider-linode/v2/linode/childaccount/tmpl"
 )
 
+func TestSmokeTests_childaccount(t *testing.T) {
```

Comment:
we will need to add new smoke tests in this function for each test file to allow `make smoke-test` pick up the new tests